res/cfg/default.toml

```diff
@@ -26,6 +26,7 @@ mc__first_epoch_number = 0
 mc__epoch_duration_millis = 86400000
 mc__first_slot_number = 0
 mc__slot_duration_millis = 1000
+allow_non_ssl = true
```

Comment:
Let's set it to `false` as default - we could enable it by default only in the `dev` config - wdyt @gilescope ?